src/main/java/uk/gov/hmcts/reform/professionalapi/controller/response/ProfessionalUsersEntityResponse.java

```diff
@@ -10,6 +11,9 @@
 
 @NoArgsConstructor
 public class ProfessionalUsersEntityResponse {
+
+    @JsonProperty
+    private String organisationIdentifier;

```

Comment:
do we required organisationIdentifier in both classes? (super and sub classes)

Reply:
There is no parent-child relationship here

Reply:
ok.